pkg/packer/packer.go

```diff
@@ -116,5 +117,21 @@ func (p *Packer) IDs(encrypted bool) (map[string]string, error) {
 		}
 	}
 
+	if len(imageIDByName) == 0 && !encrypted {
+		version := version.CleanVersion()
+		p.log.Warn("no built images found")
+
+		image, err := p.tarmak.Cluster().Environment().Provider().DefaultImage(version)
+		if err != nil {
+			return nil, err
+		}
+
+		imagesChangeTime[image.BaseImage] = image.CreationTimestamp.Time
+		imageIDByName[image.BaseImage] = image.Name
+
+		p.log.Warnf("EBS is unencrypted so using Jestack's pre-built default image from version %s",
```

Comment:
Spelling of jetstack

Reply:
Good catch, not the best thing to misspell...